apps/src/aichat/views/chatMessage.module.scss

```diff
@@ -0,0 +1,42 @@
+@import 'color.scss';
+
+.name {
+    font-family: "Gotham 5r", sans-serif;
+}
+
+.message {
+    background-color: $lightest_gray;
+    padding: 10px 12px;
+}
+
+.userMessageContainer {
+    margin: 0 0 25px 50px;
+    border-radius: 10px;
```

Comment:
Not sure why `border-radius` here and below are not being set by this stying. Any advice is appreciated~

Reply:
It's the child that has the background colour, so this radius will only do something if the overflow is hidden, but the quotes around the hidden are preventing it from working.  

That said, because this parent is larger than the child, you're not really getting the rounding exactly where you want it.  It seems more appropriate to move the border radius to the child (currently known as `#chat-workspace-message-body`) and then no hidden overflow is needed.

Reply:
Thanks so much! This makes a lot of sense. Moved the border radius styling to `.message`.
I also removed the user name according to the Figma design.

![Screen Shot 2023-08-01 at 9 31 27 AM](https://github.com/code-dot-org/code-dot-org/assets/107423305/663b4618-0f74-474b-b9d3-d0b2e4a74992)